nautobot/extras/templates/extras/dynamicgroup_edit.html

```diff
@@ -12,19 +13,99 @@
+                                    {% if child.errors %}
+                                        {% for error in child.errors.values %}{{ error }}{% endfor %}
+                                    {% endif %}
+                                {% endfor %}
+                            </div>
+                        </div>
+                    {% endif %}
+                    {{ children.non_field_errors }}
+                    <div class="form-group{% if children.errors %} has-error{% endif %}">
+                        <table class="table" id="children">
+                            {{ children.management_form }}
+                            {% for child_form in children.forms %}
+                                {% if forloop.first %}
+                                    <thead>
+                                        <tr>
+                                            {% for field in child_form.visible_fields %}
+                                                <th>{{ field.label|capfirst }}</th>
+                                            {% endfor %}
+                                        </tr>
+                                    </thead>
+                                {% endif %}
+                                <tr class="formset_row-{{ children.prefix }}">
+                                    {% for field in child_form.visible_fields %}
+                                        <td>
+                                            {% if forloop.first %}
+                                                {% for hidden in child_form.hidden_fields %}
+                                                    {{ hidden }}
+                                                {% endfor %}
+                                            {% endif %}
+                                            {{ field }}
+                                            {% if field.errors %}
+                                                <ul>
+                                                    {% for error in field.errors %}
+                                                        {# Embed an HTML comment indicating the error for extraction by tests #}
+                                                        <!-- FORM-ERROR {{ field.name }}: {{ error }} -->
+                                                        <li clas="text-danger">{{ error }}</li>
+                                                    {% endfor %}
+                                                </ul>
+                                            {% endif %}
+                                        </td>
+                                    {% endfor %}
+                                </tr>
+                            {% endfor %}
+                        </table>
+                    </div>
+                </div>
+            </div>
         </div>
     </div>
     {% include 'inc/extras_features_edit_form_fields.html' %}
 {% endblock form %}
+
+{% block javascript %}
```

Comment:
Though I don't think it's actually required with the present template hierarchy for `object_edit.html`, let's get in the habit of including a `{{ block.super }}` whenever we have a custom `{% block javascript %}`

Reply:
We should add a lint check for this TBH